### PR TITLE
make changes according to bard asks

### PR DESCRIPTION
**Pull request by chejennifer**
- enable SCATTER results for bard. e.g., "correlation of heart disease and poverty in USA" returns https://paste.googleplex.com/4888613947441152
- update the returned data_csv for TABLE results to match the ranking tile results shown on explore. Examples:
    - "poorest countries by GDP" returns https://paste.googleplex.com/6041875832635392
    - "richest countries by GDP" returns https://paste.googleplex.com/4912165551603712
    - "richest and poorest countries by GDP" returns https://paste.googleplex.com/5871962497220608

### CODE REVIEW

File: static/js/components/tiles/sv_ranking_units.tsx [hunk 1]
```diff
@@ -216,6 +215,33 @@ export function getRankingUnitTitle(
   return formatString(title, rs);
 }
 
+/**
+ * Gets the top points and bottom points to display in a ranking unit
+ * @param rankingMetadata the RankingTileSpec to get the points for
+ * @param isHighest whether or not this ranking unit is showing highest
```
Comment (juliawu) [inline static/js/components/tiles/sv_ranking_units.tsx:221]: nit: "...is showing highest" reads like it cuts too early. "...showing highest points"?

File: static/js/components/ranking_unit.tsx [hunk 2]
```diff
@@ -93,26 +93,28 @@ function getRank(
 
 // Gets list of list of points to show where each list will be separated by ...
 // and each point will have the rank set.
-function getPointsList(props: RankingUnitPropType): RankingPoint[][] {
+export function getPointsList(
+  topPoints: RankingPoint[],
+  bottomPoints: RankingPoint[],
+  isHighest: boolean,
+  numDataPoints?: number
+): RankingPoint[][] {
```
Comment (juliawu) [inline static/js/components/ranking_unit.tsx:101]: Nit: Can you add a comment doc above the function of what these parameters mean?

**Review by juliawu — APPROVED** [review]
Nice!

File: static/js/components/tiles/sv_ranking_units.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/components/tiles/sv_ranking_units.tsx]: done

File: static/js/components/ranking_unit.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/components/ranking_unit.tsx:108]: done

**Comment by chejennifer** [comment]
thanks for the review!

**Pull request merged**